libs/components/forms/src/lib/modules/input-box/input-box.component.scss

```diff
@@ -287,6 +279,12 @@ sky-input-box {
         }
 
         .sky-control-help {
+          /*
+          The help button should be above the input element to ensure it is clickable.
+          Select inputs have a z-index of 3 so that clicking the label area still opens the select.
+          */
+          z-index: 4;
+          height: 0;
```

Comment:
While using `height: 0` works, its existence isn't self-explanatory (i.e. we'd need to put a comment above this line of code explaining why it's here). 

The extra height to the field is coming from the `3px` margin-top, correct? Could we instead remove the `height: 0` and apply `-3px` to the bottom margin to make the solution self-explanatory?
```
margin: 3px 0 -3px $sky-margin-inline-compact;
```

Reply:
Updated.